sqlitecluster/SQLiteNode.cpp

```diff
@@ -2774,3 +2823,25 @@ void SQLiteNode::_sendStandupResponse(SQLitePeer* peer, const SData& message) {
     }
     _sendToPeer(peer, response);
 }
+
+void SQLiteNode::_dieIfForkedFromCluster() {
+    size_t quorumNodeCount = 0;
+    size_t forkedFullPeerCount = 0;
+    for (const auto& p : _peerList) {
+        if (!p->permaFollower) {
+            forkedFullPeerCount++;
```

Comment:
Shouldn't this be `quorumNodeCount`?

Reply:
I think they're the same thing?

Reply:
I think Ioni is right, this should be increasing `quorumNodeCount` and below inside `if (p->forked) {` we increase `forkedFullPeerCount`

Reply:
I think Ioni is right, this should be increasing `quorumNodeCount` and below inside `if (p->forked) {` we increase `forkedFullPeerCount`